src/config.rs

```diff
@@ -35,6 +35,12 @@ pub struct Discv5Config {
     /// The maximum number of established sessions to maintain. Default: 1000.
     pub session_cache_capacity: usize,
 
+    /// The one-time session timeout. Default: 30 seconds.
+    pub one_time_session_timeout: Duration,
+
+    /// The maximum number of established one-time sessions to maintain. Default: 100.
+    pub one_time_session_cache_capacity: usize,
+
```

Comment:
Since these config values are user-facing, we should probably explain here/somewhere what one time sessions are and what they do

Reply:
yep good idea